### ISSUE TITLE
Errors thrown by `golangci-lint`

### ISSUE THREAD

**Issue by echarrod** [go-playground/validator#964]
- [X] I have looked at the documentation [here](https://pkg.go.dev/github.com/go-playground/validator/v10#section-documentation) first?
- [X] I have looked at the examples provided that may showcase my question [here](/_examples)?

### Package version eg. v9, v10: 
v9.31.0

### Issue, Question or Enhancement:
When using the golangci-lint watcher provided by GoLand:
![image](https://user-images.githubusercontent.com/1381991/175047654-424af8d4-eb53-4941-9f17-f53d7f25ad50.png)
Every time a file is saved, I get errors from the validator package:
```bash
mypath/my_file.go:245:14: undeclared name: `validator` (typecheck)
	validate := validator.New()
```
and also:
```bash
mypath/my_file.go:16:2: "gopkg.in/go-playground/validator.v9" imported but not used (typecheck)
	"gopkg.in/go-playground/validator.v9"
```
For each one of the files I am using the package.

I looked under the hood and can see it runs the following command:
```golangci-lint run --disable=typecheck```

### Code sample, to showcase or reproduce:
https://goplay.tools/snippet/b--sydpv40H
```go
package main

import (
	"github.com/gofrs/uuid"
	"gopkg.in/go-playground/validator.v9"
)

type MyRequest struct {
	UserID uuid.UUID `json:"user_id" validate:"required"`
}

func main() {
	request := &MyRequest{UserID: uuid.Must(uuid.NewV4())}

	validate := validator.New()
	err := validate.Struct(request)
	if err != nil {
		panic(err)
	}
}
```

Save file, then run:
```
golangci-lint run --disable=typecheck yourfile.go
```

**Comment by hjwk** [comment]
Are you sure your import path is correct ? 

Instead of "gopkg.in/go-playground/validator.v9",  _"github.com/go-playground/validator/v10"_ is the path used in the examples and it is also the one referenced on https://pkg.go.dev/github.com/go-playground/validator/v10

**Comment by echarrod** [comment]
Hi, I believe this was the older recommended way of importing the package. Since v10, I know they have `github.com/go-playground/validator/v10` on the docs. I also tried with this import, and get the same result

**Comment by hjwk** [comment]
have you tried to run your code ? Because it looks like the validator package was simply not downloaded and thus golangci-lint is unable to find a declaration for the validator object and throws the error/warning

**Comment by echarrod** [comment]
Yes, the code runs fine, but just when the watcher runs intermittently I get this error. It behaves functionally fine, as in it builds and runs, but this output just gets returned during the watcher run
